### PR TITLE
PEP8 confirmations

### PR DESCRIPTION
**Pull request by tushar-rishav**

### CODE REVIEW

File: scorer/app.py [hunk 1]
```diff
@@ -13,39 +13,43 @@
...
+            matchChoice = getUserInput(matches)
         except KeyboardInterrupt:
             exitApp()
-        if(matchChoice == len(matches) -1 ):
+        if(matchChoice == len(matches) - 1):
             logger.debug("User chose quit")
             exitApp()
-        logger.debug("User's choice: {} {}".format(matchChoice, matches[matchChoice-1]))
+        logger.debug("User's choice: {} {}".format(matchChoice,
+                                                   matches[matchChoice-1]))
```
Comment (neo1691) [inline scorer/app.py]: @tushar-rishav Is this change necessary? Can you link me to something saying this conforms to PEP8

Comment (tushar-rishav) [inline scorer/app.py]: Sure! A maximum of 79 chars. 
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

**Comment by neo1691** [comment]
Agreed, I think I was talking about the extra line on the next line. @tushar-rishav Any reasons for closing this?

**Comment by tushar-rishav** [comment]
Well, I thought that you think that the conformations are not necessary! Shall i send a PR again? :sweat_smile:

**Comment by neo1691** [comment]
Yup! Thank You. I will look into the new PR and get back to you. Hopefully by the end of the day.

**Comment by tushar-rishav** [comment]
Cool!
